fixtures/user/user_fixture.py

```diff
@@ -123,18 +123,7 @@
 }
 '''
 
-change_user_role_mutation_response = {
-    "data": {
-        "changeUserRole": {
-            "user": {
-                "name": "Peter Walugembe",
-                "roles": [{
-                    "role": "Admin"
-                }]
-            }
-        }
-    }
-}
+change_user_role_mutation_response = "Role changed but email not sent"
```

Comment:
Good work @daniellamarr . This line is repeated in `fixtures/user_role/user_role_fixtures.py`  line `113`

Reply:
Yes, the test was written for an existing feature and for some reasons a change in my feature affected the implementation thereby making the tests fail, so I had to make the changes.